### DISCUSSION TITLE
How to make an unblocker.

### DISCUSSION THREAD

**Discussion by gs1009** [3kh0/ext-remover#567] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How do you make your own unblocker so that the school does not find it on your github and block it? Please add a video tutorial. Thanks!

**Comment by cooltoo37** [comment, marked as answer]
Hi today I will show you how to make a proxy your own very proxy watch the vid also the process is easy not hard  but takes a few good minutes for it here is vid link https://www.kapwing.com/videos/6412aa3d7615600018e95a0e if blocked use this proxy website https://example3-9hj4.onrender.com
 Thanks to the @https://github.com/interstellarnetwork group for the proxy and making it possible for us to do this.

**Reply by cooltoo37** [reply]
Your welcome

**Comment by siouxxsies** [comment]
basically deploy whatever you want like uh womginx, incognito or holy unblocker to websites like railway, glitch, replit (sometimes the github is blacklisted and you won't be able to deploy it) most of the time you'll need to signup for those websites and i recommend you do that on your own time/ at home on a different device because it involves you entering a credit card or scanning a QR code with a certain type of app.

**Reply by cooltoo37** [reply]
no with render

**Reply by cooltoo37** [reply]
just need to sign up no credit card neeed it

**Reply by siouxxsies** [reply]
> just need to sign up no credit card neeed it

i never used render so i didn't know

**Reply by siouxxsies** [reply]
> just need to sign up no credit card neeed it

thats bs lol

**Reply by siouxxsies** [reply]
> just need to sign up no credit card neeed it

render literally asks you for a credit card before deploying anything

**Reply by amelia-bartlett** [reply]
> > just need to sign up no credit card neeed it
> 
> render literally asks you for a credit card before deploying anything

just selfhost it from home then
